src/main/java/duke/commands/DeadlineCommand.java

```diff
@@ -0,0 +1,45 @@
+package duke.commands;
+
+import java.time.LocalDate;
+
+import duke.Storage;
+import duke.TaskHandler;
+import duke.exceptions.ChatBotException;
+
+
+
+/**
+ * Represents a deadline task command line
+ */
+public class DeadlineCommand extends ChatBotCommand {
+    private String taskName;
+    private LocalDate deadline;
+
+    /**
+     * Constructor
+     * @param taskName String to store the task name.
+     * @param deadline LocalDate to store the deadline.
+     */
+    public DeadlineCommand(String taskName, LocalDate deadline) {
+        this.taskName = taskName;
+        this.deadline = deadline;
+    }
+
+    /**
+     * Returns message for adding a deadline tasks.
+     *
+     * @param th TaskHandler.
+     * @param storage Storage to write and read file.
+     * @return Command response.
+     * @throws ChatBotException if encounters error.
+     */
+    public String runTask(TaskHandler th, Storage storage) throws ChatBotException {
+        th.addDeadlineTask(taskName, deadline);
+        storage.writeToFile(th);
+        String output = String.format("Got it. I've added this task: \n%s\nNow you have %d task(s) in the list",
+                th.getTaskList().get(th.getLength() - 1).toString(),
+                th.getLength()
+        );
```

Comment:
I think the ending brace should be with the last line of the statement, instead of being on its own new line. I noticed the same issue in several other places too.